core/src/main/java/org/infinispan/configuration/parsing/CacheParser.java

```diff
@@ -723,10 +730,11 @@ private void parseSegmentedCacheAttribute(ConfigurationReader reader, int index,
             break;
          }
          case CONSISTENT_HASH_FACTORY: {
-            if (reader.getSchema().since(11, 0)) {
```

Comment:
Should we have this in an else block? I guess it probably isn't that important as I doubt people were overriding the hash factory though.